ui/pages/permissions-connect/snaps/snap-update/snap-update.js

```diff
@@ -148,14 +146,6 @@ export default function SnapUpdate({
               textAlign={TEXT_ALIGN.CENTER}
             >
               {t('snapUpdateRequest', [
-                <Text
-                  as={ValidTag.Span}
-                  key="1"
-                  variant={TextVariant.bodyMd}
-                  fontWeight={FontWeight.Medium}
-                >
-                  {originMetadata?.hostname}
```

Comment:
Are we sure we want to strip this? I would assume we only strip the origin when the origin is MM?

cc @eriknson